src/DependencyInjection/PimcoreCustomerManagementFrameworkExtension.php

```diff
@@ -39,14 +40,20 @@ class PimcoreCustomerManagementFrameworkExtension extends ConfigurableExtension
      */
     public function prepend(ContainerBuilder $container)
     {
-        if ($container->hasExtension('doctrine_migrations')) {
-            $loader = new YamlFileLoader(
-                $container,
-                new FileLocator(__DIR__ . '/../Resources/config')
-            );
+        $loader = new YamlFileLoader(
+            $container,
+            new FileLocator(__DIR__ . '/../Resources/config')
+        );
 
+        if ($container->hasExtension('doctrine_migrations')) {
             $loader->load('doctrine_migrations.yml');
         }
+
+        if ($container->hasExtension('pimcore_custom_reports')) {
+            $loader->load('custom_reports.yml');
```

Comment:
```suggestion
            $loader->load('custom_reports.yaml');
```
just wondering, should we start recommending `yaml` for 11 related stuff?